extensions/wikia/Forum/DiscussionsRedirectHelper.class.php

```diff
@@ -0,0 +1,82 @@
+<?
+class DiscussionsRedirectHelper {
+	/**
+	 * @return bool
+	 */
+	public static function areForumsArchivedAndDiscussionsEnabled() {
+		return true;
+//		global $wgArchiveWikiForums, $wgEnableDiscussions;
+//
+//
+//		return $wgArchiveWikiForums && $wgEnableDiscussions;
+	}
+
+	private static function discussionsUrl() {
+		return '/d';
+	}
+
+	private static function categoryUrl( $categoryId ) {
+		if ( !empty( $categoryId ) ) {
+			return '/d/f/' . $categoryId;
+		} else {
+			return self::discussionsUrl();
+		}
+	}
+	private static function postUrl( $postId ) {
+		if ( !empty( $categoryId ) ) {
+			return '/d/p/' . $postId;
+		} else {
+			return self::discussionsUrl();
+		}
+	}
+
+	private static function getDiscussionsUrl($namespace, $id) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1142)